src/select/SimpleSelect.js

```diff
@@ -578,68 +564,62 @@ class SimpleSelect extends Component {
   }
 
   renderArrow(isOpen) {
-    const ArrowComponent = this.props.arrowComponent
+    const { arrowComponent } = this.props
+    const ArrowComponent = arrowComponent
     const { arrowRenderer } = this.props
 
-    return (<ArrowComponent
-      arrowRenderer={arrowRenderer}
-      onArrowClick={this.onArrowClick}
-      onArrowTouchEnd={this.onArrowTouchEnd}
-      isOpen={isOpen}
-    />)
+    return (
+      <ArrowComponent
+        arrowRenderer={arrowRenderer}
+        onArrowClick={this.onArrowClick}
+        onArrowTouchEnd={this.onArrowTouchEnd}
+        isOpen={isOpen}
+      />
+    )
   }
 
   renderClear() {
-    const ClearComponent = this.props.clearComponent
-    const {
-      clearable,
-      clearRenderer,
-      disabled,
-      value
-    } = this.props
+    const { clearComponent } = this.props
+    const ClearComponent = clearComponent
+    const { clearable, clearRenderer, disabled, value } = this.props
     const hasValue = Array.isArray(value) ? value.length : value
 
     if (!clearable || !hasValue) {
       return null
     }
 
-    return (<ClearComponent
-      clearRenderer={clearRenderer}
-      disabled={disabled}
-      onClearClick={this.onClearClick}
-      onClearTouchEnd={this.onClearTouchEnd}
-    />)
+    return (
+      <ClearComponent
+        clearRenderer={clearRenderer}
+        disabled={disabled}
+        onClearClick={this.onClearClick}
+        onClearTouchEnd={this.onClearTouchEnd}
+      />
+    )
   }
 
   renderLoading() {
-    const LoadingComponent = this.props.loadingComponent
+    const { loadingComponent } = this.props
```

Comment:
same nit: I would combine the `this.props` lines

Reply:
Fixed!